code/cmdline/cmdline.cpp

```diff
@@ -1843,6 +1847,14 @@ bool SetCmdlineParams()
 		}
 	}
 
+	if (vr.found()) {
+		Cmdline_enable_vr = true;
+		if (fov_arg.found() || fov_cockpit_arg.found()) {
+			Warning(LOCATION, "FoV Command line settings found. VR-Mode will override these with the HMD's native FoV...");
```

Comment:
I wonder if it makes sense to just ignore the fov arguments, given how much more relevant they are in a VR context. I know nothing about VR, so this is a genuine question: Shouldn't we be using the headset FOV exclusively?

Reply:
Yes, this is exactly what happens.
In VR mode, once the headset is initialized a fair bit later, the FOV setting will be overridden by what the API reports.
It just warns the user here, so that they don't get confused why the setting doesn't change anything

Reply:
argh, sorry, misread the code :D